build.psm1

```diff
@@ -696,19 +690,12 @@ function New-PSOptions {
     }
 
     if (-not $Runtime) {
-        $RuntimeFromDotNet = dotnet --info | ForEach-Object {
+        $Runtime = dotnet --info | ForEach-Object {
             if ($_ -match "RID") {
                 $_ -split "\s+" | Select-Object -Last 1
```

Comment:
@TravisEz13 which of the two options you prefer? 
1. Do the conversion from Win10-x64 -> Win7-x64 in build.psm1
2. Make -Runtime a mandatory parameter?